commons/src/main/java/gov/cms/qpp/conversion/DocumentationReference.java

```diff
@@ -2,13 +2,14 @@
 
 public enum DocumentationReference {
 
+	CPC_PLUS_SUBMISSIONS(14),
 	IDENTIFIERS(15),
+	PERFORMANCE_PERIOD(17),
 	CLINICAL_DOCUMENT(19),
 	PRACTICE_SITE_ADDRESS(25),
+	DOCUMENTATION_OF_TIN_NPI(28),
 	REPORTING_PARAMETERS_ACT(80),
-	MEASURE_IDS(88),
-	PERFORMANCE_PERIOD(17),
-	CPC_PLUS_SUBMISSIONS(14);
+	MEASURE_IDS(88);
 
 	private static final String BASE_PATH = "https://ecqi.healthit.gov/system/files/eCQM_QRDA_EC-508_0.pdf#page=";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)